tests/main/interfaces-kernel-module-load/task.yaml

```diff
@@ -10,6 +10,10 @@ environment:
 prepare: |
     "$TESTSTOOLS"/snaps-state install-local $SNAP_NAME
 
+restore: |
```

Comment:
nitpick this could be a defer in the prepare section instead so it's only executed if the install in the prepare succeeds (i.e. if the install fails with your branch, we will still see awkwardness from spread because the prepare fails, and then the restore also fails)

Reply:
ah that's right I had forgotten

Reply:
this makes my suggestion _even more_ of a nitpick then :-)